ui/public/index.css

```
@@ -36,6 +36,9 @@
   --trading-outgoing: #ef6194;
   --trading-incoming: #11bea9;
 
+  --subscape: #e4eeee;
+  --subscape-background: #315A5A;
```

Let's stay consistent with lowercase hex values here

```suggestion
  --subscape-background: #315a5a;
```